tests/e2e.sh

```diff
@@ -30,8 +38,8 @@ dex() {
 }
 
 deploy() {
-    $KUBECTL apply -f https://raw.githubusercontent.com/coreos/kube-prometheus/master/manifests/setup/prometheus-operator-0servicemonitorCustomResourceDefinition.yaml
-    $KUBECTL apply -f https://raw.githubusercontent.com/coreos/kube-prometheus/master/manifests/setup/prometheus-operator-0prometheusruleCustomResourceDefinition.yaml
+    $KUBECTL apply -f https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/main/manifests/setup/0servicemonitorCustomResourceDefinition.yaml
```

Comment:
We recently had to fix this in the main Observatorium/Observatorium repository too. To prevent this from failing again, it would be best practice to pin to a specific commit instead of using `main`. Can we do that here too?

Reply:
sure. updated.